calculator/calculator/CalculatedPage/CaculatedPage.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  CaculatedPage.swift
+//  calculator
+//
+//  Created by Byeon jinha on 2022/05/20.
+//
+
+import SwiftUI
+
+struct CaculatedPage: View {
+   
+    @EnvironmentObject var CalculatorDataSheet : CalculatorDataSheet
```

Comment:
```suggestion
    @EnvironmentObject var CalculatorDataSheet: CalculatorDataSheet
```

공백은 `:`롤론 뒤에만 추가되는 것이 Swift 스타일 가이드에 적절해요 : )

Reply:
:) 수정했습니다!